RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -6,9 +6,9 @@
     applyPedConstraint    = cms.bool(True),
     applyTimeConstraint   = cms.bool(True),
     applyPulseJitter      = cms.bool(False),
-    applyTimeSlew         = cms.bool(True),           #units
-    ts4Min                = cms.double(0.),           #fC
-    ts4Max                = cms.vdouble(100.,45000.), #fC # this is roughly 20 GeV
+    applyTimeSlew         = cms.bool(True),       #units
+    ts4Min                = cms.double(0.),       #GeV, for run2-2016 0 fC used
+    ts4Max                = cms.vdouble(20.,20.), #GeV, for run2-2016 100 fC used
```

Comment:
Since we are changing the meaning of the cut, maybe it's also time to name it appropriately tsTrigMin and tsTrigMax
(or ts45Min and ts45Max). [a painful step wrt HLT support though].
The fact that 4 means "sum of 4 and 5" suggests that the naming is not the most intuitive.